src/ui/map.js

```diff
@@ -25,7 +25,6 @@ import { RGBAImage } from '../util/image';
 import { Event, ErrorEvent } from '../util/evented';
 import { MapMouseEvent } from './events';
 import TaskQueue from '../util/task_queue';
-import webpSupported from '../util/webp_supported';
```

Comment:
Looks like the recent `webpSupported` changes got accidentally stripped, probably on a rebase or something?

Reply:
whoops yep, the original PR was branched off an earlier `master`, and i copied it straight from there. fixing